deployments/kustomize/image-builds/postgres.yaml

```diff
@@ -26,7 +26,9 @@ spec:
   successfulBuildsHistoryLimit: 1
   strategy:
     dockerStrategy:
-      dockerfilePath: Dockerfile
+      from:
```

Comment:
interesting. this must be related to all the image builds you removed yesterday.

Is this an image build though?

It looks like the strategy is DockerImage not DockerFile - that tells me its not getting built at all.

am I understanding this correctly?

Reply:
* this is an image build
* This means that it replaces the FROM statement in the Dockerfile with this image, therefore the only place you need to update images are at the image builds.